packages/react/menu/src/Menu.stories.tsx

```diff
@@ -13,9 +15,10 @@ import {
 import { css } from '../../../../stitches.config';
 import { foodGroups } from '../../../../test-data/foods';
 
-import type * as Polymorphic from '@radix-ui/react-polymorphic';
-
-export default { title: 'Components/Menu', excludeStories: ['TickIcon', 'styledComponents'] };
+export default {
+  title: 'Components/Menu',
+  excludeStories: ['TickIcon', 'styledComponents', 'classes'],
```

Comment:
We had a `Classes` story because we export `classes`. Just fixed that while I was here.